src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -45,6 +51,7 @@ class PillSelector extends React.Component<Props, State> {
         inputProps: {},
         placeholder: '',
         selectedOptions: [],
+        showRoundedPills: false,
```

Comment:
Is this needed or does the falsy `undefined` value suffice? That way unaffected component snapshots don't need to be updated.